personalities/sctfe/ct_server_gcp/main.go

```diff
@@ -67,6 +67,10 @@ var (
 	bucket             = flag.String("bucket", "", "name of the bucket to store the log in")
 	spannerDB          = flag.String("spanner_db_path", "", "projects/{projectId}/instances/{instanceId}/databases/{databaseId}")
 	rootsPemFile       = flag.String("roots_pem_file", "", "Path to the file containing root certificates that are acceptable to the log. The certs are served through get-roots endpoint.")
+	rejectExpired      = flag.Bool("reject_expired", false, "if true then the certificate validity period will be checked against the current time during the validation of submissions. This will cause expired certificates to be rejected.")
```

Comment:
nit: there's a mix of initial capital and initial non-cap on the flag descriptions here, would be cool to make them all be the same :)

Reply:
I fixed them all.